src/window/hotkey_config.c

```diff
@@ -283,10 +285,54 @@ static void handle_input(const mouse *m, const hotkeys *h)
     }
 }
 
+static const uint8_t *hotkey_action_name_for(hotkey_action action)
+{
+    const uint8_t *name = 0;
+    for (int i = 0; i < (NUM_VISIBLE_OPTIONS + scrollbar.max_scroll_position); i++) {
+        hotkey_widget *widget = &hotkey_widgets[i];
+        if (widget->action == action) {
+            if (widget->name_translation != TR_NONE) {
+                name = translation_for(widget->name_translation);
+            } else {
+                name = lang_get_string(widget->name_text_group, widget->name_text_id);
+            }
+            break;
+        }
+    }
+    // QUESTION
+    // is it OK return NULL pointer and crash game if we did not find "name"?
```

Comment:
Answer: yes. If we reuse the "plain_message_dialog" instead of having a custom dialog, like I suggested in another comment, it will simply not display the message when this function returns NULL.